res/matrix-org/invite_template.eml

```diff
@@ -1,25 +1,25 @@
-Date: %(date)s
-From: %(from)s
-To: %(to)s
-Message-ID: %(messageid)s
-Subject: %(subject_header_value)s
+Date: {{ date }}
+From: {{ from }}
+To: {{ to }}
+Message-ID: {{ messageid }}
+Subject: {{ subject_header_value }}
 MIME-Version: 1.0
 Content-Type: multipart/alternative;
-	boundary="%(multipart_boundary)s"
+	boundary="{{ multipart_boundary }}"
 
---%(multipart_boundary)s
+--{{ multipart_boundary }}
 Content-Type: text/plain; charset=UTF-8
 Content-Disposition: inline
 
 Hi,
 
-%(sender_display_name)s %(bracketed_verified_sender)shas invited you into a room
-%(bracketed_room_name)son Matrix. To join the conversation, either pick a
```

Comment:
huh, why is there not a space before `on`? This isn't your fault seemingly, wonder how that got there.

Would be worth fixing it in the Jinja2 version of the template also

Reply:
(or is it intentional? Same for `has` on the line above.... hm)

Reply:
If you have some spare time, mind looking into whether that's on purpose for some reason?

Reply:
Okay I cannot tell you why it's that way but I can tell you that it needs to be that way to not render with a weird extra space. I checked that on both the regular string formatting and using Jinja and the way that it is, without the space, is the way that works.

Reply:
that's good enough, as long as we know it's not going to say '@reivilibre:matrix.orghas invited you into a room My Roomon Matrix' :D